SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -119,6 +127,15 @@ public let DEFAULT_DISK_CACHE_MAX_SIZE_BYTES: Int64 = 25 * 1024 * 1024
      Enable NetworkInstrumentation span creation for https calls.
      */
     @objc public var networkInstrumentation: Bool = true
+     /**
+     Threshold, in milliseconds, from which to count a rendered frame as slow.
+     */
+    @objc public var slowFrameDetectionThresholdMs: Double = 16.7
+
+    /**
+     Threshold, in milliseconds, from which to count a rendered frame as frozen.
+     */
+    @objc public var frozenFrameDetectionThresholdMs: Double = 700
```

Comment:
Can this be a `let`, or is the value changed at some point?

Reply:
Options structures are designed to support the use case of creating one, setting various fields manually, and then passing it to the `initialize`.  We keep a copy internally so it won't change over the lifespan of the app.